articles/hooks/guides/use-the-post-user-registration-extensibility-point.md

```diff
@@ -1,28 +1,22 @@
 ---
-title: Using the Post-User Registration Extensibility Point
-description: The post-user-registration extensibility point for use with Hooks
+description: How to use the post-user-registration extensibility point
 beta: true
-toc: true
 topics:
     - hooks
     - extensibility-points   
 contentType:
   - how-to
 useCase: extensibility-hooks
 ---
+# Use the Post-User Registration Extensibility Point
```

Comment:
The same comment applies to this one. The title should include the result or goal of doing this.